styles/github/catppuccin.user.css

```diff
@@ -501,6 +501,23 @@
     --shadow-floating-legacy: 0px 6px 12px -3px @crust, 0px 6px 18px 0px @crust;
     --outline-focus: @blue solid 2px;
 
+    /* Customer Agreements */
+    .enterprise-customer-agreement {
+      color: @accent-color !important;
```

Comment:
Shouldn't this be `text` by default and only accent on hover?

Reply:
My mistake, will correct.